docs/update_and_migration/from_4.6/update_to_5.0.md

```diff
@@ -0,0 +1,1062 @@
+        symfony/web-profiler-bundle:^7.3 \
+    ;
+    ```
+
+#### Remove 4.6 LTS Updates constraints
+
+4.6 LTS Update packages are included by default in 5.0.
+Remove them from your composer.json to avoid updating their version manually with each update.
+
+For example, the following command removes all of the released LTS Updates for 4.6 from `composer.json`:
+
+```bash
+composer remove --no-update \
+    ibexa/connector-ai \
+    ibexa/connector-openai \
+    ibexa/product-catalog-date-time-attribute \
+    ibexa/product-catalog-symbol-attribute \
+    ibexa/discounts \
+    ibexa/discounts-codes \
+;
+```
+
+#### Remove PHP 8.2 error handler
+
+If you were using the [`Php82HideDeprecationsErrorHandler`](update_from_4.6.md#v468) to avoid deprecation messages,
+you must remove it:
+
+```bash
+composer config --unset extra.runtime.error_handler
+```
+
+#### Update required packages
+
+It's time to apply the new composer.json and update the dependencies:
+
+```bash
+composer update --with-all-dependencies --no-scripts
+```
+
+#### Remove Stimulus bootstrap
+
+To help moving from Symfony's Webpack Encore bundle 1.x to 2.x,
+delete the Stimulus bootstrap file
+and reset Webpack Encore recipe:
+
+```bash
+rm assets/bootstrap.js
+composer recipes:install symfony/webpack-encore-bundle --reset --force --yes
+```
+
+Compare with your previous version, and merge and test customization if needed.
+
+#### Apply [[= product_name =]] recipe
+
+=== "[[= product_name_headless =]]"
+
+    ```bash
+    composer recipes:install ibexa/headless --reset --force --yes
+    ```
+
+=== "[[= product_name_exp =]]"
+
+    ```bash
+    composer recipes:install ibexa/experience --reset --force --yes
+    ```
+
+=== "[[= product_name_com =]]"
+
+    ```bash
+    composer recipes:install ibexa/commerce --reset --force --yes
+    ```
+
+#### Sort commands
+
+Executing the recipes appends a new command at the end`composer.json`'s `auto-scripts` section, resulting in incorrect script order.
+You have to manually sort the commands so the `tsconfig.json` file
+is created by `yarn ibexa-generate-tsconfig`
+before being used by `ibexa:encore:compile`.
+Your `auto-scripts` entry should look like this:
+
+```json
+        "auto-scripts": {
+            "cache:clear": "symfony-cmd",
+            "assets:install %PUBLIC_DIR%": "symfony-cmd",
+            "yarn install": "script",
+            "yarn ibexa-generate-tsconfig --relative-paths": "script",
+            "ibexa:encore:compile --config-name app": "symfony-cmd",
+            "bazinga:js-translation:dump %PUBLIC_DIR%/assets --merge-domains": "symfony-cmd",
+            "ibexa:encore:compile": "symfony-cmd",
+            "ibexa:encore:compile --frontend-configs-name ibexa,internals,libs,richtext": "symfony-cmd"
+        },
+```
+
+#### Remove Ibexa Icons
+
+Remove from your `config/bundles.php` the line about `IbexaIconsBundle`.
+
+#### Post update script
+
+```bash
+rm -rf var/cache
+composer run-script post-update-cmd
+```
```

Comment:
I think running `post-update-cmd` might break at this point and we can consider removing it - but we can wait for Support's feedback with that